tests/phpstan/bootstrap.php

```diff
@@ -0,0 +1,100 @@
+<?php
```

Comment:
I'm not sure where best to put this, but static analysis is testing so I don't think it's an inappropriate location. Suggestions welcome though.

Reply:
I think this belongs in the tests directory since I view static analysis as a form of testing (which seems to jive with @johnbillion since he said it just above me). I view automated testing as anything that automates a process of ensuring code quality.